BookPlayer/Player/PlayerManager.swift

```diff
@@ -772,6 +774,19 @@ extension PlayerManager {
       self.audioPlayer.seek(to: CMTime(seconds: newPlayerTime, preferredTimescale: CMTimeScale(NSEC_PER_SEC)))
     }
   }
+  
+  func handleSticky(_ item: PlayableItem) {
```

Comment:
```suggestion
  func handleStickyTimer() {
```
No need to pass in the `PlayableItem`